docs/make.jl

```diff
@@ -0,0 +1,28 @@
+using ChainRulesDeclarationHelpers
+using Documenter
+using DocThemeIndigo
+using Markdown
```

Comment:
why is this needed?

Reply:
No idea.
I copied it from the ChainRulesOverloadGeneration docs.

Reply:
Answer: it isn't